src/initializers/bull/index.ts

```diff
@@ -14,6 +14,9 @@ export default async (config, orkaOptions: Partial<OrkaOptions>) => {
   const prefix = orkaOptions.appName;
   const defaultOptions = config.bull.queue.options;
   const redis = config.bull.redis || config.redis;
+  const options = defaults(omit(config.bull.redis || config.redis.options, ['url', 'tls']), {
```

Comment:
No need for defaults here, the spread operator would suffice, no?

```suggestion
  const options = {...omit(config.bull.redis || config.redis.options, ['url', 'tls']), enableReadyCheck: false};
```

Reply:
Your suggestion does not equal the `defaults` behaviour because it will always set `enableReadyCheck: false`.

The intention here is to set as default the `enableReadyCheck: false`, but to be overridden if needed via the configuration

Reply:
So?:

```
const options = {enableReadyCheck: false., ..omit(config.bull.redis || config.redis.options, ['url', 'tls'])};
```